external/fv3fit/fv3fit/reservoir/_reshaping.py

```diff
@@ -14,27 +14,11 @@ def concat_inputs_along_subdomain_features(a, b):
     return np.concatenate([a, b], axis=1)
 
 
-def stack_data(tensor, keep_first_dim: bool):
-    # Used to reshape a subdomains into a flat columns.
-    # Option to keep first dim, used in the case where time is the first dimension
-    if keep_first_dim is True:
-        n_samples = tensor.shape[0]
-        return np.reshape(tensor, (n_samples, -1))
-    else:
-        return np.reshape(tensor, -1)
-
-
-def split_1d_samples_into_2d_rows(
-    arr: np.ndarray, n_rows: int, keep_first_dim_shape: bool
-) -> np.ndarray:
+def split_1d_samples_into_2d_rows(arr: np.ndarray, n_rows: int) -> np.ndarray:
```

Comment:
Since there's no more use case for these helper functions that involves a time dim, they can be simplified or deleted.